dump/dump.go

```diff
@@ -143,6 +149,10 @@ func (d *Dumper) Dump(w io.Writer) error {
 		args = append(args, "--master-data")
 	}
 
+	if d.gtidPurged == "auto" {
+		args = append(args, fmt.Sprintf("--set-gtid-purged=%s", d.gtidPurged))
```

Comment:
don't support MariaDB now?

Reply:
Yes, this PR add GTID replication for MySQL after myqldump is done, MariaDB can still use file-pos replication.